cassandra/components.py

```diff
@@ -560,6 +567,132 @@ def run_component(self):
 
         return 0
 
+    def prep_for_xanthos(self, xanthos_dir, monthly_data):
+        """Convert a fldgen output array to Xanthos' expected input format.
+
+        Retrieve Xanthos grid cells from alternately indexed vectors.
+
+        Convert spatial data from a list of numeric vectors (each list element is one
+        month) to the input format expected by Xanthos (land cell x months).
+
+        params:
+          xanthos_dir: Full file path to xanthos input/output directory
+          monthly_data: Input data for xanthos as numpy array (cells x months)
+
+        returns:
+          2d array of Xanthos cells by month
+
+        """
+
+        # Load reference file mapping Xanthos cell index to lat/lon
+        xcells_path = os.path.join(xanthos_dir, 'input/reference/coordinates.csv')
+        if not os.path.exists(xcells_path):
```

Comment:
I believe the accepted practice in python is to simply attempt to read the input file and let the IOError exception be raised from the I/O functions if the file doesn't exist.  See, for example:
https://stackoverflow.com/a/16138864